### PR TITLE
Accessibility Guidelines - add to docs

### PR DESCRIPTION
**Pull request by MagdalenaHenke**
[Clubhouse](https://app.clubhouse.io/blocks/story/46453/add-accessibility-checklist-to-website)
[Google Doc with (almost) the same text](https://docs.google.com/document/d/1aCc1X66CR-GZS80jbZUoq_UuUIPCzj9BxlGtXzjSX-M/edit)

...yeah doing this in html is not theeee funnest. 
I'm rooting for progress over perfection, and I'm sure these guidelines will be iterated on.

Also to note: I think the line height and paragraph styling on the docs pages isn't the easiest to read. I didn't want to mess with that now because it's a pretty substantial change, but I think at some point it'd be great to have a designer help us make things look smoooooth.

Check out http://localhost:8080/help/accessibility

### CODE REVIEW

File: packages/blocks-docs/src/css/page.styl [hunk 1]
```diff
@@ -109,15 +109,24 @@ div[role="group"][tabindex] {
...
 }
 
+.help-text-list
+  list-style-type: disc;
+
+.help-text-checklist
+  li:before
+    content: "\2751"
+    position: relative
+    left: -0.5rem
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/css/page.styl]: I wanted this to have checklist feeling so this is to make the bullets visually appear more like checkable squares.
Screen Shot 2020-01-22 at 7 06 16 PM

File: packages/blocks-docs/src/data/navigation.json [hunk 2]
```diff
@@ -83,9 +83,13 @@
           "href": "/help/newblock"
         },
         {
-          "name": "Writing guidelines",
+          "name": "Writing Guidelines",
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/data/navigation.json]: felt inconsistent to have Browser *Support* but not capitalize Guidelines here.

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+            Keyboard interactions for common widgets should, as much as
+            applicable, follow the ones outlined in the{" "}
+            <a href="https://www.w3.org/TR/wai-aria-practices-1.1/">
+              ARIA Authoring Practices
+            </a>
+            .
+          </li>
+        </ul>
+
+        <h2 className="blx-h3">
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/accessibility.js]: I'm overriding the h2 styles with h3 looking ones and make the h3s look like h4s. I want a semantically correct heading structure but usual h2 styling is so big in this context to me it looks overwhelming so I made them smaller.

File: packages/blocks-docs/src/pages/help/browsersupport.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import React from "react"
 import Layout from "../../components/layout"
 import Header from "../../components/header"
 
-const WritingGuidelinesPage = () => (
+const BrowserSupport = () => (
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/browsersupport.js]: unrelated rename - noticed this copy/paste error

File: packages/blocks-docs/src/pages/help/documentation-updates.js [hunk 5]
```diff
@@ -3,15 +3,17 @@ import React from "react"
 import Layout from "../../components/layout"
 import Header from "../../components/header"
 
-const WritingGuidelinesPage = () => (
+const DocumentationUpdates = () => (
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/documentation-updates.js]: same here, I think the old naming was just copy and paste

File: packages/blocks-docs/src/pages/help/documentation-updates.js [hunk 6]
```diff
@@ -3,15 +3,17 @@ import React from "react"
...
   <Layout>
     <Header title="How to update the docs"></Header>
     <div className="l-flex-vertical doc-section">
       <div className="help-text-container">
         <h2 className="blx-h3">Everyone</h2>
-        <p className="blx-with-margin">
+        <div className="blx-with-margin">
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/documentation-updates.js]: turns out < p >s aren't _supposed_ to contain stuff like li s, so I made them divs.

File: packages/blocks-docs/src/pages/help/documentation-updates.js [hunk 7]
```diff
@@ -3,15 +3,17 @@ import React from "react"
...
           <ul className="help-text-list">
-            <li>Follow the <a href="/help/writing">Blocks Writing Guidelines</a></li>
+            <li>
```
Comment (MagdalenaHenke) [inline packages/blocks-docs/src/pages/help/documentation-updates.js]: since I opened the file it prettierd it so this is just whitespace changes

**Comment by MagdalenaHenke** [comment]
@runigoswami - how about we do something fun and tag you in this one? :) 

The text is pretty close to what I updated here: https://docs.google.com/document/d/1aCc1X66CR-GZS80jbZUoq_UuUIPCzj9BxlGtXzjSX-M/

But if you'd like to "practice" reviewing on github, you could leave comments here! You can check out the review app here: https://deploy-preview-205--awesome-knuth-6f8b93.netlify.com/help/accessibility

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+          </li>
+          <li>
+            <h3 className="blx-h4">Content</h3>
+            <ul className="help-text-checklist">
+              <li>
+                All content follows the{" "}
+                <a href="https://blocks.cbrebuild.com/help/writing">
+                  Blocks writing guidelines
+                </a>
+                —write as simply as possible, use gender neutral language when
```
Comment (runigoswami) [inline packages/blocks-docs/src/pages/help/accessibility.js]: sentence might be simplified (ironic!) to read:
"All content follows the Blocks writing guidelines – write simply and use inclusive language."

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 9]
```diff
@@ -0,0 +1,275 @@
...
+              </li>
+              <li>
+                All user actions receive swift success or failure feedback,
+                error states are also programmatically connected to their
+                inputs.
+              </li>
+            </ul>
+          </li>
+          <li>
+            <h3 className="blx-h4">Content</h3>
```
Comment (runigoswami) [inline packages/blocks-docs/src/pages/help/accessibility.js]: nitpick but technically Images (item #2) would also fall under Content 

consider either combining this section with the images section (maybe preferred) OR renaming this section to be "Writing" or "Copy"

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 10]
```diff
@@ -0,0 +1,275 @@
...
+          <li>
+            Free axe Chrome extension (
+            <a href="https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd">
+              axe devtool
+            </a>
+            )
+          </li>
+        </ul>
+
+        <h2 className="blx-h3">What’s missing / Future work to be done</h2>
```
Comment (runigoswami) [inline packages/blocks-docs/src/pages/help/accessibility.js]: This a long section heading – maybe "Next steps" or "Future work"

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 11]
```diff
@@ -0,0 +1,275 @@
...
+        <ul className="help-text-list">
+          <li>Set up guidelines around responsiveness.</li>
+          <li>
+            Define clearer areas of responsibility and workflows for
+            incorporating accessibilty into the whole development lifecycle.
```
Comment (runigoswami) [inline packages/blocks-docs/src/pages/help/accessibility.js]: typo: accessibility

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 12]
```diff
@@ -0,0 +1,275 @@
...
+            Questions to answer are, for example:
```
Comment (runigoswami) [inline packages/blocks-docs/src/pages/help/accessibility.js]: not sure if we need to include these open questions here – they seem much more granular in scope than we necessarily need to publish

Comment (MagdalenaHenke) [reply packages/blocks-docs/src/pages/help/accessibility.js]: I'll take them out!

File: packages/blocks-docs/src/pages/help/accessibility.js [hunk 13]
```diff
@@ -0,0 +1,275 @@
...
+          Shortened accessibility reference for BlocksDS
+        </h2>
+        <p className="blx-with-margin">
+          This is a subset of the WCAG standards and minimum standards to check
+          for whenever making BlocksDS changes. It’s not meant as a most
+          comprehensive checklist, but to provide non-overwhelming reference in
+          place:
+        </p>
+
+        <ol>
```
[the remark this answers is not in the document]
Comment (MagdalenaHenke) [reply packages/blocks-docs/src/pages/help/accessibility.js]: Maybe the answer is that I shouldn't have made them a list at all, and headings are enough.

File: packages/blocks-docs/src/css/page.styl (hunk 1 above)
Comment (tariqporter) [reply packages/blocks-docs/src/css/page.styl]: @MagdalenaHenke I'm not sure how this should be handled when rebasing for turning these files to markdown https://github.com/floored/blocks/pull/201

**Comment by lkim2** [comment]
Hi hi, I know you've merged this already, but I have some feedback: 

1. Use title case on all section headers. Minding voice, tone, and style > Minding Voice, Tone and Style. Our writing guidelines don't say anything about this yet, but titles should be title case for Header 1 and 2. https://blog.apastyle.org/apastyle/2012/03/title-case-and-sentence-case-capitalization-in-apa-style.html

2. The first "Why accessibility is important" paragraph has some long sentences in there. I've been using hemingway to try to refine my copy. http://www.hemingwayapp.com/ Would love to help with the copy here if I have time before I go! Also, do we want to encourage people to feed accessibility updates back into blocks?

3. Page title says "blocks by CBRE Build" > "Accessibility"

4. Other possible things to include: States - focus, active, default, disabled, hover, empty, error

**Comment by MagdalenaHenke** [comment]
@lkim2 Thank you!

1. I can fix the title case stuff soon!
2. I am no copy expert at all, and very, very, very happy to just take any copy you throw at me!
3. Fixed that! Now our title updates on all pages. Was a different PR that I opened after writing these docs. :P
4. Happy about more states, don't think I fully understand the UI indication (is that about how all feature need to be reachable through the UI?). The zoom stuff I'm very for - we're breaking a lot by not dealing well with text resize, too, for example. I left it out because it felt like a very undefined thing that needed more specific guidance.

**Comment by lkim2** [comment]
1. 👍 
2. 👌 
3. 🏅 
4. Not necessarily that all things must be accessible through UI, but that you as a designer should think through "have I thought about keyboard, mouse, trackpad, touch, and UI access". I agree with zooming. Maybe that can be added to future stuff. Maybe they can both be added to the future section. "Accessibility for additional inputs (trackpad, mouse, etc.) and Responsive / Zoom"

**Comment by MagdalenaHenke** [comment]
@lkim2 Liz you also are welcome to make copy and title case changes by yourself right through git if you'd like! (Or we can do it together through git.)

**Comment by lkim2** [comment]
O true!

Get Outlook for Android<https://aka.ms/ghei36>
________________________________
